pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -904,9 +923,15 @@ const storeSyslogTemplate = `{{- define "storeSyslog" -}}
 	  message_key {{.PayloadKey}}
 	</format>
 {{end -}}
+{{- if .Hints.Has "pick_daemon_name" }}
+  <buffer $.systemd.u.SYSLOG_IDENTIFIER>
+    @type file
+    path '{{.BufferPath}}_journal'
```

Comment:
Same comment as above for BufferPath https://github.com/openshift/cluster-logging-operator/blob/b2ff008f366e01658f77a083c49a235bb2ee5e36/pkg/generators/forwarding/fluentd/fluent_conf.go#L111

Reply:
Fixed it.